app/samples/udp_bcast/src/main.cpp

```diff
@@ -17,19 +17,27 @@
 LOG_MODULE_REGISTER(main);
 
 K_MSGQ_DEFINE(broadcast_queue, sizeof(const char[32]), 10, 4);
+K_MSGQ_DEFINE(other_broadcast_queue, sizeof(const char[16]), 10, 4);
 
 int main() {
     static constexpr char ipAddrStr[] = "10.0.0.0";
     static constexpr int udpPort = 10000;
-    static constexpr char broadcastStr[] = "Hello, Launch!";
+    static constexpr char broadcastStr[32] = "Hello, Launch!";
+    static constexpr char otherBroadcastStr[16] = "Hello, World!";
     static constexpr int broadcastStrLen = sizeof(broadcastStr);
+    static constexpr int otherbroadcastStrLen = sizeof(otherBroadcastStr);
```

Comment:
should we not just put this at the top and use it in all the places where we neeed this length (line 20, 26, 33)?

Reply:
Done